packages/core/strapi/tests/api/relations.test.e2e.js

```diff
@@ -0,0 +1,690 @@
+              myCompo: {
+                compo_products_ow: { [connectOrSet]: oneRelation },
+                compo_products_mw: { [connectOrSet]: manyRelations },
+              },
+            },
+            populateShop
+          );
+
+          expect(shop).toMatchObject({
+            attributes: {
+              myCompo: {
+                compo_products_mw: { data: [{ id: id1 }, { id: id2 }] },
+                compo_products_ow: { data: { id: id1 } },
+              },
+              products_mm: { data: [{ id: id1 }, { id: id2 }] },
+              products_mo: { data: { id: id1 } },
+              products_mw: { data: [{ id: id1 }, { id: id2 }] },
+              products_om: { data: [{ id: id1 }, { id: id2 }] },
+              products_oo: { data: { id: id1 } },
+              products_ow: { data: { id: id1 } },
+            },
+          });
+        });
+
+        test('In reversed order', async () => {
+          const oneRelation = mode === 'object' ? [{ id: id1 }] : [id1];
+          const manyRelations = mode === 'object' ? [{ id: id1 }, { id: id2 }] : [id1, id2];
+          manyRelations.reverse();
+
+          const shop = await createEntry(
+            'shops',
+            {
+              name: 'Cazotte Shop',
+              products_ow: { [connectOrSet]: oneRelation },
+              products_oo: { [connectOrSet]: oneRelation },
+              products_mo: { [connectOrSet]: oneRelation },
+              products_om: { [connectOrSet]: manyRelations },
+              products_mm: { [connectOrSet]: manyRelations },
+              products_mw: { [connectOrSet]: manyRelations },
+              myCompo: {
+                compo_products_ow: { [connectOrSet]: oneRelation },
+                compo_products_mw: { [connectOrSet]: manyRelations },
+              },
+            },
+            populateShop
+          );
+
+          expect(shop).toMatchObject({
+            attributes: {
+              myCompo: {
+                compo_products_mw: { data: [{ id: id2 }, { id: id1 }] },
+                compo_products_ow: { data: { id: id1 } },
+              },
+              products_mm: { data: [{ id: id2 }, { id: id1 }] },
+              products_mo: { data: { id: id1 } },
+              products_mw: { data: [{ id: id2 }, { id: id1 }] },
+              products_om: { data: [{ id: id2 }, { id: id1 }] },
+              products_oo: { data: { id: id1 } },
+              products_ow: { data: { id: id1 } },
+            },
+          });
+        });
+      });
+    }
+  );
+
+  describe('Update an entity relations', () => {
+    describe.each([
+      ['directly in the array ([3])', 'object'],
+      ['an object in the array ([{ id: 3 }])', 'array'],
+    ])('ids being %s', (name, mode) => {
+      test('Adding id3', async () => {
+        const createdShop = await createEntry(
+          'shops',
+          {
+            name: 'Cazotte Shop',
+            products_ow: { connect: [id1] },
```

Comment:
We only have documented the array format [{id: 3}], should we document this other one too?

Reply:
yes! I think it is a clean way to connect when you do not care about order.